src/writing-policies/go/05-e2e-tests.md

```diff
@@ -0,0 +1,212 @@
+# End-to-end testing
```

Comment:
For later: I have the impression we can reuse this section for all languages. On each language we explain how to obtain the Wasm binary, and from that point on, e2e can be achieved with the same strategy for all of them.